.conform.yaml

```diff
@@ -12,7 +12,7 @@ policies:
           gitHubOrganization: siderolabs
       spellcheck:
         locale: US
-      maximumOfOneCommit: true
```

Comment:
this expected?

Reply:
yep, that's to simplify further backports if any